<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGeneratePlaces">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать стартовые позиции</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerateWeights">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать довесы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerateResults">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать итоговые результаты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChooseCurrentPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Указать путь к файлу с участниками</string>
    </property>
   </widget>
   <widget class="QLabel" name="textCurrentPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий путь:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textChosenCurrentPath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите файл с участниками</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listParticipants">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textParticipants">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Участники</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="menuProgramActionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="menuHelpActionOpenHelp"/>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuProgramActionQuit">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="menuHelpActionOpenHelp">
   <property name="text">
    <string>Открыть справку</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
